app/scripts/utils/debounce.js

```diff
@@ -9,21 +11,24 @@
  * Once webpack 2 with tree-shaking is supported I'd advocate to use lodash's
```

Comment:
We are no longer using webpack, and I'm thinking we could replace this with [`just-debounce-it`](https://github.com/angus-c/just).